src/CarbonAware.WebApi/test/integrationTests/IntegrationTestingBase.cs

```diff
@@ -1,4 +1,7 @@
 ﻿using CarbonAware.DataSources.Configuration;
+using CarbonAware.DataSources.Json.Mocks;
+using CarbonAware.DataSources.Mocks;
+using CarbonAware.DataSources.WattTime.Mocks;
```

Comment:
Do you actually need these new `using` calls in this file?

Reply:
Yes.  At least the way the namespaces are structured right now.

```
using CarbonAware.DataSources.Json.Mocks; // This gives us access to JsonDataSourceMocker
using CarbonAware.DataSources.Mocks; // This gives us access to the interface we use to define the _dataSourceMokcer member variable
using CarbonAware.DataSources.WattTime.Mocks; // This gives us access to WattTimeDataSourceMocker
```